webgui/src/flipmouse/js/tabslot.js

```diff
@@ -35,13 +83,95 @@ tabSlot.initSlots = function () {
     });
 };
 
+tabSlot.initProcesses = function () {
+	if(inElectron === false) {
+		var resultList = ["not available / nicht verfügbar"];
+		//add the process list to the GUI
+		L('.process-select').forEach(function (elem) {
+			elem.innerHTML = L.createSelectItems(resultList);
+		});
+		return;
+	}
+	
+	storage.setDataPath(require('os').homedir());
+	console.log("Looking for config json here: " + storage.getDataPath());
+	
+	//get an active process if one is set for this slot
+	storage.get('FLipMouseGUIConfiguration',function(error, processActivationList) {
+		//if we got a list back (JSON file exists & contains data)
+		if (typeof processActivationList !== 'undefined') {
+			//find active slot in list
+			var index = processActivationList.findIndex(function(item, i){
+				 return item.slot === flip.getCurrentSlot()
+			});
+			activateOnProcess = processActivationList;
+			//if index found -> set name of process
+			if(index !== -1) active = processActivationList[index]["process"];
+			else active = "";
+		} else active = "";
```

Comment:
I dont't find the place where `active` is defined. Maybe use `let active = ''` above the if statement, then you also do not neeed this `else` here.